src/purchases.ts

```diff
@@ -171,6 +174,13 @@ export default class Purchases {
    */
   public static VERIFICATION_RESULT = VERIFICATION_RESULT;
 
+  /**
+   * Enum of StoreKit version.
+   * @readonly
+   * @enum {string}
+   */
+  public static STOREKIT_VERSION = STOREKIT_VERSION;
```

Comment:
I think we need to do the same with the `PURCHASES_ARE_COMPLETED_BY` enum. So consumers can access that from `react-native-purchases` and not the internal library.